### ISSUE TITLE
Empty save result even though it shows up in the menu

### ISSUE THREAD

**Issue by ItsEverest** [mircokroon/minecraft-world-downloader#479]
**Describe the bug**
When I try to download the save the chunks show up in the preview although when I save it and try to preview it in mineways or in game it's just empty world

**Steps to reproduce**
Connect to any server running on 1.19.2 / 1.19.3 with world downloader on

**Log output**
  
**Screenshots**
![image](https://user-images.githubusercontent.com/78271762/214371992-11a679cd-de4b-4fd3-8e6d-18c57f05ffd2.png)
![image](https://user-images.githubusercontent.com/78271762/214372029-15b7f524-5fc4-4c7a-acdb-f9bc248092f0.png)
![image](https://user-images.githubusercontent.com/78271762/214372153-87f07885-7a1b-4a7b-b9b8-a3944fad3f72.png)

**Version**
 - Game: Minecraft 1.19.2
 - Downloader: latest
 - Launcher (if login issues): Multimc

**Comment by mircokroon** [comment]
Is this still an issue using the latest version?

**Comment by elblasy1986** [comment]
Yes, I have the latest version and facing the exact same issue with one server"not all" and I have filled a bug issue for my problem.
